src/v3/src/transformer/i18n/transformSecondEmailInputExplain.ts

```diff
@@ -37,6 +37,7 @@ export const transformSecondEmailInputExplain: TransformStepFnWithOptions = ({
         element,
         name: 'bottomExplain',
         i18nKey: 'oie.profile.additional.secondemail.subtitle',
+        replacementTokens: { '<$1>': '<span class="strong">', '</$1>': '</span>' },
```

Comment:
So I started writing this when you had the replacementFn. Now that you updated this, I think most of my concerns are gone but I'm wondering if we can make it even more concise?

I.e.:
```
add Translation({
....,
arguments: {
  $1:  { 
element: span,
class: "no-translate"
}
  }
}
})

```

Reply:
I like this, we can definitely do this.

Reply:
I am afraid that using `.replace` on the `$1` alone could be dangerous. I was considering using regex to only find `$1` that is wrapped with brackets but then it would also require me to handle the open tag differently than the closing tag. 

Also, I think this logic will be short lived, we were discussing today how we can get away from using these tags/html embedded in translation strings because it is a sort of anti-pattern.

Reply:
I think one potential solution might be to match /(<`argument`>)(.*)(</`argument`>)/, and then do some magic with matching groups. 

Otherwise perhaps the logic here might help? : https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/handlebars/helper-i18n.ts#L14

However, that might be a bit complex for now and I agree if this is not going to be the long term solution, I'm fine with revisiting it another day :)

Reply:
All set! Thank you so much for the recommendation!